File: ManagerIngestTag/ManagerIngestTag/Models/TicketIngestModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ManagerIngest.Models
{
    public class TicketIngestModel
    {
        public Guid TicketIngestId { get; set; }
        public string Name { get; set; }
        public string CreateName { get; set; }
        public string TopicName { get; set; }
        public string ProgramName { get; set; }
        public string CameramanName { get; set; }
        public string ProductionName { get; set; }
        public string ReporterName { get; set; }
        public DateTime SaveDocument { get; set; }
        public bool IsReporting { get; set; }
        public bool IsNew { get; set; }
        public bool IsCategory { get; set; }
        public bool IsOtherProgram { get; set; }
        public string StatusIngest { get; set; }
    }
    public class TicketIngestFull : TicketIngestModel
    {
        public string StatusName { get; set; }
        public List<IngestDetailFull> IngestDetailFull { get; set; }
    }
}

File: MangagerInsestTag.FE/src/app/shared/TicketIngest/ticket-ingest.model.ts
import { IngestDetail } from "../IngestDetail/ingest-detail.model";

export class TicketIngest {
    TicketIngestId: string  = '00000000-0000-0000-0000-000000000000';
    Name: string = '';
    CreateName: string = '';
    TopicName: string = '';
    ProgramName: string = '';
    CameramanName: string = '';
    ProductionName: string = '';
    ReporterName: string = '';
    SaveDocument: string = '';
    IsReporting: boolean = false;
    IsNew: boolean = false;
    IsCategory: boolean = false;
    IsOtherProgram: boolean = false;
    StatusIngestCode: string = '';
    StatusIngest: string = '';
    DateCreate: Date = new Date();
}
export class TicketIngestFull extends TicketIngest{
    IngestDetailFull: IngestDetail[] = [];
}